README.md

```diff
@@ -27,11 +28,17 @@ for Java, with the following changes (mandatory):
 
 * IntelliJ
   - Go to "Preferences" and them to "Coding Style" in IntelliJ.
```

Comment:
Could you rephrase this as in the Eclipse setting: `Go to "Preferences" -> "Coding Style"` and remove IntelliJ, to have consistency.